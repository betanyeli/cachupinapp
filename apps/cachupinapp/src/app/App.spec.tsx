import React from 'react';
import { render } from '@testing-library/react-native';

import App from './App';

describe('App.tsx', () => {

  it('Should generate a snapshot', () => {
    const { toJSON } = render(<App />);
    expect(toJSON).toMatchSnapshot();
  });
  it('Should render correctly', () => {
    const { getByText } = render(<App />);
    expect(getByText('Cachupinapp')).toBeTruthy();
  });
});

