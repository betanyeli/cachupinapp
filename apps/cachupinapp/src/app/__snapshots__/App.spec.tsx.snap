// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.tsx Should generate a snapshot 1`] = `[Function]`;
